docker/build.sh

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env sh
 set -eux
 
-apt-get update
```

Comment:
we're doing it earlier in the dockerfile now with the new patch

Reply:
didn't need to do this at all in fact, and doing it was what was causing needless http requests